create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -387,25 +406,38 @@ private boolean isTitleOriginal(TitletextTp titletextTp) {
         return extractHead().getCitationTitle().getTitletext();
     }
 
-    private List<Contributor> generateContributorsFromAuthorGroup(AuthorGroupTp authorGroupTp) {
+    private List<Contributor> generateContributorsFromAuthorGroup(AuthorGroupTp authorGroupTp,
+                                                                  PersonalnameType correspondencePerson) {
         return authorGroupTp.getAuthorOrCollaboration()
             .stream()
-            .map(this::generateContributorFromAuthorOrCollaboration)
+            .map(o -> generateContributorFromAuthorOrCollaboration(o, correspondencePerson))
```

Comment:
What is an 'o'?

Reply:
It's the circle around your head 👼 (120 line limit makes me invent names like 'o').